callbacks/utils/SaveWholeSlices.py

```diff
@@ -201,32 +197,31 @@ def __call__(
             )
 
             sample_image = self.save_image(
-                image_path=self.image_dataset[sample_idx]["target_path"],
+                image_path=image_sample["target_path"],
                 image_type="target",
-                image=self.image_dataset[sample_idx]["target"],
+                image=image_sample["target"],
             )
 
             # Only save these images if the segmentation mask isn't black
             # We expect the model to generate black segmentation crops,
             # which will present regardless of weather or not the whole segmented image
```

Comment:
```suggestion
            # which will present regardless of whether or not the whole segmented image
```
Typo before this PR?

Reply:
Good catch, I'll change this. Maybe my head was stuck in the clouds